nengo/neurons.py

```diff
@@ -7,11 +7,44 @@
 from nengo.exceptions import SimulationError, ValidationError
 from nengo.params import Parameter, NumberParam, FrozenObject
 from nengo.utils.compat import range
-from nengo.utils.neurons import settled_firingrate
 
 logger = logging.getLogger(__name__)
 
 
+def settled_firingrate(step_math, J, states,
+                       dt=0.001, settle_time=0.1, sim_time=1.0):
```

Comment:
Not sure why this is still in here ... shouldn't this be moved to nengo_extras?

Reply:
I think because we use it in `Izhikevich`.